packages/next/src/client/components/navigation.ts

```diff
@@ -211,7 +211,10 @@ function useSelectedLayoutSegments(
   parallelRouteKey: string = 'children'
 ): string[] {
```

Comment:
```suggestion
): string[] | null {
```

Reply:
This is not the correct place for this type change. We don't want to override the type if you only use App Router, hence we only do it in the compat types file